public/js/p3/widget/SubSystemsMemoryGridContainer.js

```diff
@@ -288,21 +323,25 @@ define([
 				function(selection, container){
 					//subsystem tab
 					if (selection[0].document_type === "subsystems_subsystem") {
-						var query = "and(in(genome_id,(" + container.state.genome_ids.join(',') + ")),in(subsystem_id,(" + selection.map(function(s){
-							return s.subsystem_id;
-						}).join(',') + ")))&select(feature_id)&limit(25000)";
+						
+						var subsystem_ids = selection.map(function(s){
+							return encodeURIComponent(s.subsystem_id)
+						});
+
+						var query = "q=genome_id:(" + container.state.genome_ids.join(" OR ") + ") AND subsystem_id:\"" + subsystem_ids.join(',') + "\"&select(feature_id)&limit(25000)";
```

Comment:
i don't think this will work with multiple subsystem ids. look at the genome id part. 
the final query should be like 
subsystem_id:( "subsystem_id_1" OR "subsystem_id_2" OR "subsystem_id_3" )